src/helpers.ts

```diff
@@ -15,6 +15,7 @@ export const baseTypeConversionMap = new Map<string, string>([
     ["object", "any"],
     ["sequence", "Array"],
     ["record", "Record"],
+    ["undefined", "void"],
```

Comment:
Web IDL has switched from `void` to `undefined`, see heycam/webidl#60 and heycam/webidl#906. Many specifications have already made the switch, so we need to support this.

For now, I'm mapping this back to TypeScript's `void` type. However, we could also move this into `sameTypes` and actually emit `undefined` in the generated type definitions. Thoughts? 🙂

Reply:
I like this change except in unions. There is one in CustomElementRegistry:

```
    get(name: string): CustomElementConstructor | void;
```

You can't use `?.` from the value returned by the accessor. This is admittedly a failing of Typescript, but `T | void` is rare type for all `T`.

Not sure whether it's easier to fix the emitter or to add an overridingType entry for the accessor

Reply:
I can add a special case in the emitter to replace `void` with `undefined` if it appears in a union type. Right now, it's just for this one method in `CustomElementRegistry`, but maybe similar methods pop up in the future for which this would apply. 🙂